spec/publisher/publishing_content_to_government_frontend_spec.rb

```diff
@@ -0,0 +1,36 @@
+feature "Publishing content from Publisher to Government Frontend", publisher: true do
+  include PublisherHelpers
+
+  let(:title) { title_with_timestamp }
+  let(:slug) { "help/" + slug_with_timestamp }
+
+  scenario "Publishing a Help guide" do
+    given_there_is_a_draft_help_guide
+    and_i_publish_it
+    then_i_can_view_it_on_gov_uk_and_was_rendered_by_government_frontend
```

Comment:
Looks like this is missing an "it" - also is it worth separating into separate `then_i_can_view_it_on_gov_uk` and `and_it_was_rendered_by_government_frontend` methods?